src/Conf.ml

```diff
@@ -741,6 +742,39 @@ let config =
     Arg.(
       value & opt list_assoc default & info ["c"; "config"] ~doc ~docs ~env)
 
+let apply_janestreet_profile c =
+  { c with
```

Comment:
I'm not sure that it makes sense to pass in a config to a profile, since they are intended to set all the values.